library/std/src/os/windows/fs.rs

```diff
@@ -471,6 +471,14 @@ pub trait MetadataExt {
     /// `fs::metadata` or `File::metadata`, then this will return `Some`.
     #[unstable(feature = "windows_by_handle", issue = "63010")]
     fn file_index(&self) -> Option<u64>;
+
+
+    /// Returns the change time, which is the last time file metadata was changed, such as 
+    /// renames, attributes, etc.
+    /// 
+    /// This will return `None` if the `Metdata` instance was not created using the FILE_BASIC_INFO type,
```

Comment:
Seems this would be better.
```suggestion
    /// Returns the change time, which is the last time file metadata was changed, such as 
    /// renames, attributes, etc.
    /// 
    /// This will return `None` if the `Metadata` instance was not created using the `FILE_BASIC_INFO` type.
```